app/models/merchant.rb

```diff
@@ -1,7 +1,8 @@
 class Merchant <ApplicationRecord
   has_many :items, dependent: :destroy
   has_many :item_orders, through: :items
-  has_many :merchant_employees
+  has_many :orders, through: :item_orders
```

Comment:
good call here, zach

Reply:
haha after an hour or two of struggle, that was fun yesterday, good troubleshooting by the whole group here